docs/src/main/asciidoc/rest.adoc

```diff
@@ -1581,6 +1581,12 @@ public User userPrivate() {
 When the result the `userPublic` method is serialized, the `id` field will not be contained in the response as the `Public` view does not include it.
 The result of `userPrivate` however will include the `id` as expected when serialized.
 
+===== Reflection-free Jackson serialization
+
+Out-of-the-box Jackson serialization converts objects into their json representation introspecting them through a heavy use of reflection. However, the general Quarkus philosophy is to avoid reflection as much as possible, often replacing it with build time code generation. Following this idea, also in this case it is possible to automatically generate at build time implementations of the Jackson `StdSerializer`, one for each class to be converted in json, that can be subsequently used by Quarkus at runtime to perform the json serialization of the objects returned by a rest endpoint without any use of reflection.
+
+This feature is turned off by default, but it can be enabled setting the configuration property `quarkus.rest.jackson.optimization.enable-reflection-free-serializers=true`.
```

Comment:
```suggestion
This feature is turned off by default, but it can be enabled by setting the configuration property `quarkus.rest.jackson.optimization.enable-reflection-free-serializers=true`.
```